widgetastic_manageiq/__init__.py

```diff
@@ -1022,6 +1022,45 @@ def workaround_save_issue(self):
         )
 
 
+class ReactCodeMirror(Widget):
+    """Looks the same as ScriptBox but it is changed in 5.11
+    https://github.com/JedWatson/react-codemirror"""
+
+    ROOT = "//div[contains(@class,'miq-codemirror')]//textarea"
+    PARENT_DIV = "./.."  # we need to change visibility of this one
+    CLICK_DIV = "./../.."  # the widget has very nested divs,we need to click on that one ¯\_(ツ)_/¯
+
+    def __init__(self, parent, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+
+    @property
+    def element(self):
+        return self.browser.element(self)
+
+    @property
+    def is_editable(self):
+        if self.element.get_attribute("contentEditable") == "true":
+            return True
+        else:
+            return False
+
+    def make_editable(self):
+        if not self.is_editable:
+            self.browser.set_attribute("contentEditable", "true", self)
+
+    def fill(self, value):
+        self.make_editable()
+        self.browser.set_attribute("style", "overflow: visible", self.PARENT_DIV)
+        self.browser.element(self.CLICK_DIV).click()
+        self.element.send_keys(value)
```

Comment:
what will happen with previous contents of this view ? 
are there any cases when we would need to clear/replace previous contents ?

Reply:
well, nothing :)  I also haven't found such cases but if there're, I may need to account for that as well